File: PenilaianMahasiswa.java
// File: PenilaianMahasiswa.java
import java.util.Scanner;

public class PenilaianMahasiswa {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);


        // 1. TIPE DATA dan INPUT

        System.out.print("Masukkan nama mahasiswa: ");
        String nama = input.nextLine(); // tipe data String

        System.out.print("Masukkan nilai Tugas: ");
        double tugas = input.nextDouble(); // tipe data double

        System.out.print("Masukkan nilai UTS: ");
        double uts = input.nextDouble();

        System.out.print("Masukkan nilai UAS: ");
        double uas = input.nextDouble();


        // 2. OPERATOR dan PERHITUNGAN
        
        double nilaiAkhir = (tugas * 0.3) + (uts * 0.3) + (uas * 0.4); // operator * dan +
        boolean lulus = nilaiAkhir >= 60; // operator perbandingan >=


        // 3. PERCABANGAN (IF - ELSE)

        System.out.println("\n=== HASIL PENILAIAN MAHASISWA ===");
        System.out.println("Nama: " + nama);
        System.out.println("Nilai Tugas: " + tugas);
        System.out.println("Nilai UTS: " + uts);
        System.out.println("Nilai UAS: " + uas);
        System.out.println("Nilai Akhir: " + nilaiAkhir);

        if (lulus) {
            System.out.println("Status: Lulus Mas Bro ");
        } else {
            System.out.println("Status: Tidak Lulus Mas Bro ");
        }


        // 4. PERULANGAN (FOR)

        System.out.println("\nDaftar percobaan pengambilan mata kuliah:");
        for (int i = 1; i <= 3; i++) {
            System.out.println("Percobaan ke-" + i);
        }


        // 5. OUTPUT AKHIR

        System.out.println("\nTetap Menyerah dan jangan semangat, " + nama + "!");

        input.close(); // menutup Scanner
    }
}

File: penilaianMahasiswa.php
<?php

echo "Masukkan nama mahasiswa: ";
$nama = readline(); // tipe data string

echo "Masukkan nilai Tugas: ";
$tugas = (float) readline(); // tipe data float (double di PHP)

echo "Masukkan nilai UTS: ";
$uts = (float) readline();

echo "Masukkan nilai UAS: ";
$uas = (float) readline();



$nilaiAkhir = ($tugas * 0.3) + ($uts * 0.3) + ($uas * 0.4); // operator * dan +
$lulus = $nilaiAkhir >= 60; // operator perbandingan >=



echo "\n=== HASIL PENILAIAN MAHASISWA ===\n";
echo "Nama: $nama\n";
echo "Nilai Tugas: $tugas\n";
echo "Nilai UTS: $uts\n";
echo "Nilai UAS: $uas\n";
echo "Nilai Akhir: $nilaiAkhir\n";

if ($lulus) {
    echo "Status: Lulus Mas Bro\n";
} else {
    echo "Status: Tidak Lulus Mas Bro\n";
}



echo "\nDaftar percobaan pengambilan mata kuliah:\n";
for ($i = 1; $i <= 7; $i++) {
    echo "Percobaan ke-$i\n";
}



echo "\nTetap Menyerah dan jangan semangat, $nama!\n";
?>
